receiver/vcenterreceiver/documentation.md

```diff
@@ -820,6 +820,21 @@ Total memory  capacity of the host system.
 | ---- | ----------- | ---------- | ----------------------- | --------- |
 | MiBy | Sum | Double | Cumulative | false |
 
+### vcenter.vm.cpu.time
+
+CPU time spent in idle, ready or idle state.
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| % | Gauge | Double |
+
+#### Attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| spent_state | Whether the CPU time was spent in idle, ready or wait state. | Str: ``idle``, ``ready``, ``wait`` |
```

Comment:
i think you have to regenerate docs even if you don't take my naming suggestion below because this is the wrong name

Reply:
I think this is the description of the attribute but its showing the wrong attribute name. Anyway, `make generate` fixes it. Also I will change it to adhere to the comment below.